test-case-reporting/src/setup_db.ts

```diff
@@ -40,9 +40,11 @@ export async function setupDb(db: Database): Promise<unknown> {
       table.foreign('build_id').references('builds.id');
     })
     .createTable('test_cases', table => {
-      table.increments('id').primary();
+      table.string('id', 32).primary();
```

Comment:
Let's include a comment that this is an md5

Reply:
Done.

Reply:
`table.string('id', 32).primary().comment('MD5 hash of the name');`
(this way it's also stored in the database itself)

Reply:
Unresolving to add the in-db comment

Reply:
Done.